seatunnel-translation/seatunnel-translation-spark/seatunnel-translation-spark-common/src/main/java/org/apache/seatunnel/translation/spark/common/serialization/InternalRowConverter.java

```diff
@@ -102,10 +100,7 @@ private static InternalRow convert(SeaTunnelRow seaTunnelRow, SeaTunnelRowType r
             if (TypeConverterUtils.ROW_KIND_FIELD.equals(rowType.getFieldName(i))) {
                 values[i].update(seaTunnelRow.getRowKind().toByteValue());
             } else {
-                Object fieldValue = convert(seaTunnelRow.getField(i), rowType.getFieldType(i));
-                if (fieldValue != null) {
-                    values[i].update(fieldValue);
-                }
+                values[i].update(convert(seaTunnelRow.getField(i), rowType.getFieldType(i)));
```

Comment:
```suggestion
                Object fieldValue = convert(seaTunnelRow.getField(i), rowType.getFieldType(i));
                if (fieldValue != null) {
                    values[i].update(fieldValue);
                }
```
you can see [Bug] [seatunnel-translation-spark] Fix NPE caused by incorrect setting of field nullable ([#2670](https://github.com/apache/incubator-seatunnel/issues/2670))